docs/PublishingGuide.md

```diff
@@ -0,0 +1,67 @@
+## Using Travis to automate publishing of your report to GitHub Pages
+
+[GitHub Pages](https://pages.github.com/) enables you to publish your report online for free.
```

Comment:
Perhaps change to `Using Travis enables you to automate the RepoSense analysis and publish the report to GitHub Pages for free.`?